pyzscaler/zia/locations.py

```diff
@@ -91,10 +99,18 @@ def list_sub_locations(self, location_id: str):
         """
         return self._get(f"locations/{location_id}/sublocations", box=BoxList)
 
-    def list_locations_lite(self):
+    def list_locations_lite(self, **kwargs):
         """
         Returns only the name and ID of all configured locations.
 
+        Keyword Args:
+            **max_items (int, optional):
+                The maximum number of items to return before stopping iteration.
+            **max_pages (int, optional):
+                The maximum number of pages to request before throwing stopping iteration.
```

Comment:
Quality/Grammar: The maximum number of pages to request before stopping iteration.